src/test/playwright/e2e/exam/ExamResults.spec.ts

```diff
@@ -0,0 +1,216 @@
+                await examParticipation.makeSubmission(exercise.id!, exercise.type!, exercise.additionalData);
+            }
+
+            await examParticipation.handInEarly();
+            await examStartEnd.pressShowSummary();
+        });
+
+        test.beforeAll('Assess student submissions', async ({ browser }) => {
+            const page = await browser.newPage();
+            const examManagement = new ExamManagementPage(page);
+            const examAssessment = new ExamAssessmentPage(page);
+            const courseAssessment = new CourseAssessmentDashboardPage(page);
+            const exerciseAssessment = new ExerciseAssessmentDashboardPage(page);
+            const exerciseAPIRequests = new ExerciseAPIRequests(page);
+
+            await Commands.login(page, tutor);
+            await startAssessing(course.id!, exam.id!, 0, 60000, examManagement, courseAssessment, exerciseAssessment);
+            await examAssessment.addNewFeedback(7, 'Good job');
+            await examAssessment.submitTextAssessment();
+            await startAssessing(course.id!, exam.id!, 1, 60000, examManagement, courseAssessment, exerciseAssessment);
+
+            const modelingExerciseAssessment = new ModelingExerciseAssessmentEditor(page);
+            await modelingExerciseAssessment.addNewFeedback(5, 'Good');
+            await modelingExerciseAssessment.openAssessmentForComponent(0);
+            await modelingExerciseAssessment.assessComponent(-1, 'Wrong');
+            await modelingExerciseAssessment.clickNextAssessment();
+            await modelingExerciseAssessment.assessComponent(0, 'Neutral');
+            await modelingExerciseAssessment.clickNextAssessment();
+            await examAssessment.submitModelingAssessment();
+            await Commands.login(page, instructor);
+            await exerciseAPIRequests.evaluateExamQuizzes(exam);
+        });
+
+        test('Check exam result overview', async ({ page, login, examAPIRequests, examResultsPage }) => {
+            await login(studentOne);
+            await page.goto(`/courses/${course.id}/exams/${exam.id}`);
+            const gradeSummary = await examAPIRequests.getGradeSummary(exam);
+            await examResultsPage.checkGradeSummary(gradeSummary);
+        });
+
+        test('Check exam text exercise results', async ({ page, login, examParticipation, examResultsPage }) => {
+            await login(studentOne);
+            await page.goto(`/courses/${course.id}/exams/${exam.id}`);
+            const exercise = exerciseArray[0];
+            await examParticipation.checkResultScore('70%', exercise.id!);
+            await examResultsPage.checkTextExerciseContent(exercise.id!, exercise.additionalData!.textFixture!);
+            await examResultsPage.checkAdditionalFeedback(exercise.id!, 7, 'Good job');
+        });
+
+        test('Check exam programming exercise results', async ({ page, login, examParticipation, examResultsPage }) => {
+            test.fail();
```

Comment:
What does this do? Because the tests in general succeed.

Reply:
That's to mark that test fails correctly. Should I leave it as a failing test so that we are aware of the issue?

There's a bug in programming exercise results on exam result page. All tasks are shown as pending even after exam results are published. I'll create an issue about that.

Reply:
If the test isn't working, I'd prefer it to get skipped.

Reply:
Okay, sounds good to me. I'll skip it with `test.fixme()`